rainbows.rb

```diff
@@ -1,3 +1,4 @@
+require "colorize"
```

Comment:
Rainbow allows you to create colors from the rgb color codes (I cheated and looked at @lisa-sano 's solution which does this)